packages/webmcp-cli-skill/domains/publish-article-in-juejin.md

```
@@ -0,0 +1,117 @@
+# 自动化掘金文章发布专家指南
+
+本指南面向 AI Agent，详细说明如何使用 `webmcp-cli` 与浏览器交互，在掘金发布文章
+
+---
+
+### 1. 详细发布步骤
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)